<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>H80Form</class>
 <widget class="QWidget" name="H80Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H80Param</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>H80 Param</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxH80">
        <property name="text">
         <string>H80 tape copy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxRemoveTail">
        <property name="text">
         <string>Remove the last option tail block</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
